src/common/FaceliftLogging.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#include <QLoggingCategory>
+
+Q_DECLARE_LOGGING_CATEGORY(FACELIFT)
+
+typedef const QLoggingCategory& (*LoggingCategoryGetter)();
```

Comment:
even though it i going to be in the facelift namespace, I would call it FaceliftLoggingCategoryGetter

Reply:
facelift::QLoggingCategory